app/assets/stylesheets/flash.scss

```diff
@@ -0,0 +1,27 @@
+.flash {
+  padding: 8px 35px 8px 14px;
+  margin-bottom: 18px;
+  color: #c09853;
+  text-shadow: 0 1px 0 rgba(255, 255, 255, 0.5);
```

Comment:
Color literals like `rgba(255, 255, 255, 0.5)` should only be used in variable declarations; they should be referred to via variable everywhere else.